cmd/config-downloader/downloader.go

```diff
@@ -65,7 +65,9 @@ func downloadFromSSM(region, parameterStoreName, mode string, credsConfig map[st
 	}
 
 	ssmClient := ssm.New(ses)
-	input := ssm.GetParameterInput{}
+	input := ssm.GetParameterInput{
+		WithDecryption: aws.Bool(true),
```

Comment:
Actually can we do and avoid the `SetName` call. Init struct in literal way is more common in go.

```go
input := ssm.GetParameterInput{
		Name: aws.String(parameterStoreName),
		WithDecryption: aws.Bool(true),
	}
```